agent/agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -211,7 +211,7 @@ func (m *PGStatMonitorQAN) Run(ctx context.Context) {
 	if current, _, err := m.monitorCache.getStatMonitorExtended(ctx, m.q, normalizedQuery); err == nil {
 		m.monitorCache.refresh(current)
 		m.l.Debugf("Got %d initial stat monitor.", len(current))
-		running = true
+		running = true // nolint: ineffassign
```

Comment:
I think that linter warning actually highlights the problem. That variable overwritten on line 255.